src/walletSdk/Horizon/Stellar.ts

```diff
@@ -73,4 +81,60 @@ export class Stellar {
       throw e;
     }
   }
+
+  async submitWithFeeIncrease({
+    sourceAddress,
+    timeout,
+    baseFeeIncrease,
+    operations,
+    signingAddresses,
+    baseFee,
+    memo,
+    maxFee,
+  }: SubmitWithFeeIncreaseParams): Promise<Transaction> {
+    const builder = await this.transaction({
+      sourceAddress,
+      timebounds: timeout,
+      baseFee,
+      memo,
+    });
+
+    operations.forEach((op) => {
+      builder.addOperation(op);
+    });
+
+    const transaction = builder.build();
+    if (signingAddresses?.length) {
```

Comment:
@Ifropc what's the use case for the [signer function in kotlin here](https://github.com/stellar/kotlin-wallet-sdk/blob/77e4bfe0eae19b2ede5af231147351f8367cef01/wallet-sdk/src/main/kotlin/org/stellar/walletsdk/horizon/Stellar.kt#L202)? I'm imagining it's for signing with one or multiple other keypairs, but lmk if that's wrong

Reply:
It's to sign the transaction. In general, it could be not trivial, so I left it for the developer to implement